INSTALL.md

```diff
@@ -8,6 +8,10 @@ INSTALL/UPGRADE VM
     
     > vagrant plugin install vagrant-vbguest
     
+    Update your dotfiles in your own `~` directory 
+    
+    Update the `config.dist.yaml` file as needed (primarily memory and shared files location)  There is no longer a merge between config files.
```

Comment:
You can put a config.yaml file in root and it overlaps the dist. Only set the vars you want.